### DISCUSSION TITLE
Community improvement

### DISCUSSION THREAD

**Discussion by Xyncgas** [dotnet/csharplang#7158] [Q&A] [answered]
Maybe we can intergrate ChatGPT in all issues that's a bug or a question, to help people immediately to help filter out actual difficult issues for actual human to respond saving both sides' energy thus being an improvement to the community over all

**Comment by jnm2** [comment, marked as answer]
That would be something GitHub would have to do if it was prompting within the UI of this site, but anyone could definitely do an experiment offsite and see if it yielded useful enough results for something to be made of it.

**Reply by Xyncgas** [reply]
I don't care about having all projects on github to use AI, you are correct would it be happening it's something github has to implement themselves to provide the feature, I am here for the csharplang project to perhaps introduce a bot that respond to questions and trivial things delivering a 'quick' tier of ChatGPT generated answers before the 'quality' tier of human tailored response kicks in if it's needed

**Reply by jnm2** [reply]
If it worked _really well_, I could see that. It could quickly become an annoyance even in the best scenario.

**Reply by HaloFour** [reply]
A bit of the opposite issue, but I could see there being value in a Github feature that can take a discussion/issue with a lot of comments and summarize the major points and the latest direction.

**Comment by CyrusNajmabadi** [comment]
I don't see value in this effort being specific to csharplang. If useful,. All of GitHub should get it. I'd make the suggestion to them.
